posts/2022-04-15-libertydevops-gitjenkinshelmiks.adoc

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "Continuous integration and delivery (CI/CD) of Jakarta EE & MicroProfile applications with Open Liberty to Kubernetes by using Jenkins"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/shamjithantholi
+author_github: https://github.com/shamjithantholi
+seo-title: TITLE - OpenLiberty.io
+seo-description: DESCRIPTION
+blog_description: "A high level explanation about how to build a devops pipeline with Jenkins as the core controller component "
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Continuous integration and delivery (CI/CD) of Jakarta EE & MicroProfile applications with Open Liberty to Kubernetes by using Jenkins
+Shamjith Antholi https://github.com/shamjithantholi
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+[#Intro]
+== DevOps, CI/CD, Jenkins & OpenLiberty ( to deploy Jakarta EE & MicroProfile application)
+DevOps is a culture or a process, which enables the integration of the software development (dev) and IT operations (ops). It can facilitate the setup of deployment pipeline and helps to improve the speed of software delivery. Automation of the entire build, test, and deployment of the software applications can be achieved through DevOps process. There are tons of efficient applications and process methodologies available in the market that can be readily integrated to the DevOps pipeline, which can help to achieve the wanted results.
+
+CI/CD are acronym’s for continuous integration, and continuous delivery or deployment used in DevOps. Continuous integration is a process where developers frequently merge code changes into a source code repository, which automatically start code build, security scan, and test execution and deployment. Automation of this process enables organizations to release on a more frequent basis without compromising on quality. There are various tools available to automate the CI/CD process. In this blog post we'll discuss how you can use Jenkins to automate the CI/CD process for building, testing, and deploying your Jakarta EE & MicroProfile applications 
+
+Jenkins is an open source automation server, which helps to automate application code build, test, and deployment. Jenkins comes with a hundred’s of plug-ins, which eases the various DevOps pipeline tool integrations into the DevOps pipeline.
+
+Open Liberty is a lightweight Java runtime for building cloud-native applications and microservices. In this blog, it details how you can use Jenkins in CI/CD pipeline for Jakarta EE & MicroProfile application build/test/deployment (to deploy using Openliberty ) . To try Openliberty link:https://openliberty.io[OpenLiberty] 
```

Comment:
Remove this paragraph. It's not necessary. I know we've not mentioned Jakarta EE and MicroProfile yet but let's see how the whole post works then we'll look at where it's appropriate to fit those terms in. We can't just stuff terms in - it needs to work for readability too.

Reply:
Done